README.md

```diff
@@ -113,42 +113,42 @@ which is fired when a segment is skipped
     "end": 3000 // in milliseconds
   }
 }
-````
+```
 
 ### ChapterStarted
 
 which is fired when the chapters for a track are loaded
 
-````json5
+```json5
 {
   "type": "ChapterStarted",
   "op": "event",
   "guildId": "...",
   "chapter": {
     "name": "Prelude",
     "start": 0, // in milliseconds
-    "end": 0, // in milliseconds (null if last chapter => until end of track)
+    "end": 0, // in milliseconds
     "duration": "PT0S" // ISO-8601
   }
 }
-````
+```
 
 ### ChapterStarted
 
 which is fired when a new Chapter starts
-````json5
+```json5
 {
   "type": "ChapterStarted",
   "op": "event",
   "guildId": "...",
   "chapter": {
     "name": "Prelude",
     "start": 0, // in milliseconds
-    "end": 0,// in milliseconds (null if last chapter => until end of track)
-    "duration": "PT0S" // ISO-8601 (null if last chapter => until end of track)
+    "end": 0,// in milliseconds
+    "duration": "PT0S" // ISO-8601
```

Comment:
can't we provide this in milliseconds or something?, this seems really inconvenient